tests/lib/rc-parser-test.js

```diff
@@ -29,5 +29,5 @@ test('RcParser should return a json object', t => {
 })
 
 function filePath (filename) {
-  return path.join('boostnoterc', filename)
+  return path.join(`${__dirname}/boostnoterc`, filename)
```

Comment:
breaking change in ava [v0.17.0](https://github.com/avajs/ava/releases/tag/v0.17.0)